Sources/SwiftFortuneWheel/Utils/Audio/AudioPlayerManager.swift

```diff
@@ -40,12 +40,6 @@ class AudioPlayerManager {
         for _ in 0..<AudioPlayerManager.defaultStartingPlayerCount {
             players.append(createNewPlayerAttachedToEngine())
         }
-        
-        do {
-            try engine.start()
```

Comment:
If you remove the engine.start, it can cause a lag on slower devices that will make the sound play unsynchronized. However, engine.start to mute the other sound inputs, instead of engine.start, engine.prepare() should be called.